From: =?utf-8?q?Pierre-Elliott_B=C3=A9cue?= <peb@debian.org>
Date: Mon, 5 Aug 2019 11:56:56 +0200
Subject: lxc.pc.in: removes @DLOG_LIBS@ which is not expanded upon build

---
 lxc.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lxc.pc.in b/lxc.pc.in
index 5689208..0da5021 100644
--- a/lxc.pc.in
+++ b/lxc.pc.in
@@ -9,5 +9,5 @@ Description: linux container tools
 Version: @PACKAGE_VERSION@
 URL: http://linuxcontainers.org
 Libs: -L${libdir} -llxc -lutil -lpthread -ldl
-Libs.private: @CAP_LIBS@ @SECCOMP_LIBS@ @OPENSSL_LIBS@ @SELINUX_LIBS@ @PAM_LIBS@ @DLOG_LIBS@
+Libs.private: @CAP_LIBS@ @SECCOMP_LIBS@ @OPENSSL_LIBS@ @SELINUX_LIBS@ @PAM_LIBS@
 Cflags: -I${includedir}
